app/util/appUtil.js

```diff
@@ -337,14 +337,36 @@ async function getItem(tokenId) {
 
   if (tokenId) {
     await api.isReady
-    const item = await api.query.simpleNftModule.tokensById(tokenId)
+    const rawItem = await api.query.simpleNftModule.tokensById(tokenId)
```

Comment:
make it a one line `await api.query.simpleNftModule.tokensById(tokenId).toJSON()`? or even better

```js
const item = {
  ...await api.query.simpleNftModule.tokensById(tokenId).toJSON(),
  timestamp: await getTimestamp(item.created_at),
}
```